### DISCUSSION TITLE
urm

### DISCUSSION THREAD

**Discussion by Sqrts321** [community/community#140435] [New to GitHub]
Uh... I don't know what to say. I just thought it would be fun to make a random discussion. My first discussion yaaaaaay!

test:
![1 (1)](https://github.com/user-attachments/assets/18d90bf1-4703-42b0-a53c-23d5cc0a32d5)

**Comment by henry-AY** [comment]
Welcome to GitHub sqrts123!

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @sqrts123 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by Sqrts321** [reply]
ty

**Comment by ercanak** [comment]
Numaram

**Comment by Sqrts321** [comment]
e

**Comment by Sqrts321** [comment]
@iscariots are you the spammer from scratch cause if you are I have a bone to pick with you, please stop being mean to my friends imma seem so stupid if this isnt

**Reply by Sqrts321** [reply]
Hello my dear @iscariots, did I spell your name wrong? Well I don’t care. What you’ve been doing to the scratch community is unacceptable, they are my friends. They are like a family to me, and what happens when you say that stuff about my family, you’re gonna get it handed to you. Oh, I forgot to mention who I am, mainly because I‘m not involved in your doxing stalking drama, it's me squirtle/the_Squirtle321/Super_Squirtle123 and many other things I also now go by Squishy. But besides that, the acts you’ve been pulling off are childish and immature, and I want you to stop. It’s hurting the people I love, to the point where some have been wanting to end their lives. This is not okay. Calling Moth and Craz groomers for absolutely no reason is unacceptable, I will not tolerate it. Scratch is a kid’s website that is meant for all ages, because of this there are little kids on the platform and spamming your “corn” to them is just COMPLETELY wrong. This cannot be justified, I don’t know who you think you are, but you aren’t so high and mighty. Heck you make other minors do your dirty work for you, I’ve been on Scratch for 3 and a half years and I have never seen anyone as disgusting as you are. Stay away from my friends that includes every griffpatcher, everyone on pixilart and go get a fucking life. Dox me because you tooootally can, I don’t care at this point. This behavior is terrible, we are all people just like you. You are not more important than us and we are not more important than you, we are all equal so start acting that way.

While I’m here I would like to send you a few messages just for you from your… loving fans:
“Hi” -Alex
“Go to f****** h***, you creep. You’re not escaping your damn consequences, you hear me?” -Shadow
“F*** you” -Isolationism
“Hahaaa… Fuck you!” -Zaboo
“I am not a groomer and I am giving you a nice lil one-finger salute” -craz
“What’s the point of doing this?” -BiomeBash
![Screenshot 2025-05-12 4 57 08 PM](https://github.com/user-attachments/assets/a9e60f75-2ba2-40b9-b954-bdfc96a0879e)
![Screenshot 2025-05-12 4 53 17 PM](https://github.com/user-attachments/assets/cf0e4b08-f8ef-40bd-961f-24d41675158e)
![Screenshot 2025-05-12 4 53 02 PM](https://github.com/user-attachments/assets/c80a998c-7500-4ab5-9b77-b049900aedc5)
![Screenshot 2025-05-12 4 52 23 PM](https://github.com/user-attachments/assets/882fbc23-c359-41fa-8716-fa6df394e2e8)
![Screenshot 2025-05-12 4 47 07 PM](https://github.com/user-attachments/assets/2b831e2e-d40d-4ad8-8f62-653e601f75e7)
![Screenshot 2025-05-12 4 45 54 PM](https://github.com/user-attachments/assets/273c08fe-3f21-4889-ad7c-c1ca4ec6f641)
![Screenshot 2025-05-12 4 43 59 PM](https://github.com/user-attachments/assets/1575002e-3a4b-4b2b-89e4-f6dbd2bf90d0)

if you need me I'll be at www.pixilart.com/thesquirtle321/

**Reply by Sqrts321** [reply]
sorry for anyone else who's reading this and are confused

**Reply by Sqrts321** [reply]
sure

**Reply by Sqrts321** [reply]
where's your proof

**Reply by Sqrts321** [reply]
ofc not but I know you can easily get people to do things for you

**Reply by belleisntokay** [reply]
okay discord mod ahh 💀

**Reply by Sqrts321** [reply]
email who?
bitch you don't got my email

**Reply by Sqrts321** [reply]
oh right- well you don't got the one I can see *face palm*

**Reply by Sqrts321** [reply]
no shit- I just remembered

**Reply by Sqrts321** [reply]
you  don't have the email I can see though

**Reply by Sqrts321** [reply]
you better not be fucking emailing me

**Reply by belleisntokay** [reply]
the only email shown on your profile is the sqrts123 email

**Reply by Sqrts321** [reply]
oh shit-

**Comment by Sqrts321** [comment]
![Screenshot 2025-04-29 12 24 37 AM](https://github.com/user-attachments/assets/8c58851f-d223-4772-a9bb-95d265b84b1d)
me when must code

**Reply by Sqrts321** [reply]
![Screenshot 2025-05-07 6 09 35 AM](https://github.com/user-attachments/assets/92c18a73-6eb1-4b50-9a31-2ced8aced922)
POV: 1000000000 lines of code

**Reply by Sqrts321** [reply]
![peak](https://github.com/user-attachments/assets/fda20642-c521-4649-a5ff-0356133274fb)
![gigi-keep-gambling-shhh-keep-gambling](https://github.com/user-attachments/assets/fcffaa6b-4403-4478-92d6-739ab6090bdc)
